subprojects/build-cache-http/src/main/java/org/gradle/caching/http/HttpBuildCache.java

```diff
@@ -37,7 +36,6 @@
  *
  * @since 3.5
  */
-@Incubating
 public class HttpBuildCache extends AbstractBuildCache {
```

Comment:
`AbstractBuildCache` should not be incubating either.

Reply:
De-incubated.